src/frontend/typer/service.ts

```diff
@@ -17,3 +17,10 @@ export interface IOppfølgingstask {
     harMottattSvar: boolean;
     antallTilOppfølging: number;
 }
+
+export interface IAntallFeiletOgManuellOppfølging {
```

Comment:
Antall av hva da? Jeg ser ut i fra konteksten at dette er antall tasker, men det burde fremkomme av navnet på interfacet. I backend har du kalt denne for `TaskerMedStatusFeiletOgManuellOppfølging`, burde man bruke det samme navnet her?

Reply:
Synes for øvrig at `AntallTaskerMedStatusFeiletOgManuellOppfølging` blir mer presist siden dette objektet ikke er en liste med flere tasker, så min anbefaling er å bruke dette navnet både i frontend og backend

Reply:
Enig at AntallTaskerMedStatusFeiletOgManuellOppfølging er et mer presist navn, bytter til det.